client/app/src/services/network.service.js

```diff
@@ -348,22 +347,32 @@
 
     // Updates peer with all currency values relative to the USD price.
```

Comment:
>// Updates peer with all currency values relative to the USD price. 

relative to the EUR price* or could change the method name below to be more specific and remove the comment

Reply:
Ye updated the comment!